Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook

```diff
@@ -433,13 +430,7 @@
                         {
                           "columnId": "IngestionPercent",
                           "label": "Ingestion Percentage"
-                        },
-                        {
-                          "columnId": "TimeSinceLastUpdate",
-                          "label": "Usage Stats Updated"
-                        },
-                        {
-                          "columnId": "Billable"
+                        
```

Comment:
Was the Billable column removed intentionally?

Reply:
Added: columnId- Billable